packages/terra-site/src/examples/popup/PopupTopCenter.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Popup from 'terra-popup';
+import Button from 'terra-button';
+import ExamplePopupContent from './ExamplePopupContent';
+
+class PopupTopCenter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  handleOnChange() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    return (
+      <div style={{ display: 'inline-block' }} ref={this.setButtonNode}>
```

Comment:
We need to be care when we have custom styles like this in our documentation. When you have code in the docs that don't use best practices, those easily manifest their way into their consuming applications. Perhaps there is a way we can remove this?